### PR TITLE
Full rewrite

### PR DESCRIPTION
**Pull request by Jibec**
Package install was failing on YunoHost 3.0 (Debian 9), so I have it some love :)

### CODE REVIEW

**Comment by Jibec** [comment]
Thanks for the review.
Is this unmaintained thing a new rule?

**Comment by frju365** [comment]
Yes and no. See https://github.com/YunoHost-Apps/grav_ynh/issues/9

**Comment by Jibec** [comment]
OK, so this is official.
I want to make sure not to offense current maintainers, because past contributions need respect, and because I wish to have them confirm I can change the licence of the package from GPL to AGPL.

**Comment by frju365** [comment]
Yes, I agree totally, but Cyp and lithrel haven't appeared for at least 1 year... and doesn't answerthe issue.

btw: Why do you want to change package licence to AGPL ? GPL is better from my point of view...

**Comment by frju365** [comment]
Ok, I add you and @Jibec to the existing Grav Team.

File: README.md [hunk 1]
```diff
@@ -1,114 +1,45 @@
...
-3. Cliquez sur installer
-4. Rendez-vous à l'url que vous avez configuré
+## Installation
 
-Depuis la ligne de commande:
+Use either YunoHost's admin panel or command line:
 
 ```sh
-sudo yunohost app install -l grav https://github.com/YunoHost-Apps/grav_ynh
+sudo yunohost app install --url grav https://github.com/YunoHost-Apps/grav_ynh
```
Comment (tituspijean) [inline README.md]: `--url` is not listed when I do `yunohost app install -h`. I suggest the following as a minimal command:
```
sudo yunohost app install https://github.com/YunoHost-Apps/grav_ynh
```

File: README.md [hunk 2]
```diff
@@ -1,114 +1,45 @@
...
-
-- Update version
-
-### Version 1.1.1 (13/03/17)
-
-- Fix package_check [source](https://github.com/YunoHost/package_check)
-
-### Version 1.1.0 (02/03/17)
+```sh
+sudo yunohost app upgrade --url https://github.com/YunoHost-Apps/grav_ynh kanboard
```
Comment (tituspijean) [inline README.md]: Same comment.
```sh
sudo yunohost app upgrade grav -u https://github.com/YunoHost-Apps/grav_ynh
```

File: README.md [hunk 3]
```diff
@@ -1,114 +1,45 @@
...
+[ ] ARM
 
-### Configuration
+* to be added:
+[ ] send email to admin
 
-1. Start url https://domaine.tld/grav
+## Links
 
+ * Report a bug about this package: https://github.com/YunoHost-Apps/funkwhale_ynh
```
Comment (tituspijean) [inline README.md]: Copy-paste from Funkwhale? :)

**Comment by Jibec** [comment]
so, I read what @tituspijean did, I'm fine with all what it did and did the same myself.

The only difference is this line:
https://github.com/tituspijean/grav_ynh/commit/f1b8557b1e27fa122361516ecf077855b84f9643#diff-2da33fb08900b413f2f199fe4ba4aa0fL15

I'm unsure of the impact. I think Strict-Transport-Security (HSTS) is for a whole domain, enabling it without making sure grav is installed for a whole domain may have side effects on apps requiring simple http access. Was it removed for this reason?

File: README.md [hunk 4]
```diff
@@ -1,114 +1,45 @@
...
-- Fix Packaging [Source](https://forum.yunohost.org/t/news-about-app-management-and-packaging-in-yunohost-2-4/1379/1)
+# State of this package
 
-### Version 1.0.0 (11/11/15)
+* works fine:
+[x] install/remove/backup/remove/upgrade with x86_64
 
-- First version
+* to be confirmed:
+[ ] upgrade from old package to new package
```
Comment (tituspijean) [inline README.md]: We could improve the code here and below, the checkboxes are displayed on one line.

File: scripts/_common.sh [hunk 5]
```diff
@@ -0,0 +1 @@
+#!/bin/bash
```
Comment (tituspijean) [inline scripts/_common.sh]: We could delete this file?

**Comment by Jibec** [comment]
Great, thanks.
Can you merge and open a PR there? https://github.com/YunoHost/apps/blob/master/community.json

Revision: HEAD
State: working

**Comment by tituspijean** [comment]
If we opt for `HEAD`, let's also create a `testing` branch to avoid any badly written commit to break everything on the users' side. What do you think?

**Comment by Jibec** [comment]
It's fine to me, I thought it was already the case.

**Pull request merged**